lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -639,6 +638,44 @@ def deregister_inbound_handler(handler)
   attr_accessor :receiver_thread # :nodoc:
   attr_accessor :dispatcher_thread # :nodoc:
   attr_accessor :waiters # :nodoc:
+
+  @tlv_log_file = nil
+
+  def log_packet(packet, packet_type)
+    if framework.datastore['sessiontlvlogging']&.downcase == 'console'
+      log_packet_to_console(packet, packet_type)
+    elsif framework.datastore['sessiontlvlogging']&.downcase =~ /^file:.+/
```

Comment:
```suggestion
    elsif framework.datastore['sessiontlvlogging'] =~ /^file:.+/
```

If we did want case insensitivity we could use `/^file:.+/i`